src/hooks/api/types.ts

```diff
@@ -0,0 +1,30 @@
+import { useMemo } from 'react';
+
+import { useGetTermsQuery } from '@/hooks/api/terms';
+import { Scope } from '@/pages/create/EventForm';
+
+type GetThemesByEventTypeIdArguments = {
+  scope: Scope;
+};
+
+const useGetTypesByScopeQuery = (
+  { scope }: GetThemesByEventTypeIdArguments,
+  configuration: any = {},
+) => {
+  const getTermsQuery = useGetTermsQuery({
+    enabled: !!scope,
+    ...configuration,
+  });
+
+  const data = useMemo(() => {
+    if (!getTermsQuery.data?.terms) return;
+    const terms = getTermsQuery.data.terms.filter(
+      (term) => term.scope.includes(scope) && term.domain === 'eventtype',
+    );
+    return terms;
```

Comment:
We can probably just return the filter here?
What do  you think?

```suggestion
    return getTermsQuery.data.terms.filter(
      (term) => term.scope.includes(scope) && term.domain === 'eventtype',
    );
```

Reply:
https://github.com/cultuurnet/udb3-frontend/pull/407/commits/fa2f27657b6b57bfba007e795d2cfe3cf7dd101c